src/EntityGraphQL/Schema/MethodField.cs

```diff
@@ -91,17 +93,22 @@ ExecutionOptions executionOptions
                     Schema,
                     Name,
                     this,
-                    gqlRequestArgs ?? new Dictionary<string, object>(),
+                    gqlRequestArgs ?? new Dictionary<string, object?>(),
                     Arguments.Values,
                     p.ParameterType,
                     variableParameter,
                     docVariables,
                     validationErrors
                 )!;
                 allArgs.Add(argInstance);
+
+                if (p.ParameterType.BaseType == typeof(PropertySetTrackingDto))
```

Comment:
This is the magic (along with the `public class UpdateMovieArgs : PropertySetTrackingDto`) right?

If so, if you have another arg class that inherits from `UpdateMovieArgs` this will not work. Need to figure out if any of the base types are `PropertySetTrackingDto`.

Also we should be looking for `IPropertySetTrackingDto` right?